src/renderer/components/SystemStats.tsx

```diff
@@ -28,27 +28,38 @@ export const SystemStats = memo(() => {
     return (
         <HStack>
             {data &&
-                data.map((usage) => (
-                    <Tooltip
-                        borderRadius={8}
-                        key={usage.label}
-                        label={`${usage.percent.toFixed(1)}%`}
-                        px={2}
-                        py={1}
-                    >
-                        <Box key={usage.label}>
-                            <CircularProgress
-                                capIsRound
-                                color={usage.percent < 90 ? 'blue.400' : 'red.400'}
-                                size="42px"
-                                trackColor="var(--bg-700)"
-                                value={usage.percent}
-                            >
-                                <CircularProgressLabel>{usage.label}</CircularProgressLabel>
-                            </CircularProgress>
-                        </Box>
-                    </Tooltip>
-                ))}
+                data.map((usage) => {
+                    let color;
+                    if (usage.percent < 50) {
+                        color = 'green.400';
+                    } else if (usage.percent < 90) {
+                        color = 'orange.400';
```

Comment:
Request to change this to a yellow rather than an orange. I checked color-blindness filters, and while green.400 seems to have plenty of blue to stand out, the orange and red end up overly similar for certain types of colorblindness.